repos/system_upgrade/el7toel8/actors/migratesendmail/libraries/library.py

```diff
@@ -0,0 +1,41 @@
+import re
+import ipaddress
+import shutil
+
+BackupSuffix = '.bak'
+
+# false positives blacklist
+rfp = re.compile(r'(^\s*RIPv6:::1\b)|(@\s+\[IPv6:::1\]\s+>)')
+
+rs = re.compile(r'IPv6:[0-9a-fA-F:]*::[0-9a-fA-F:]*')
+
+try:
+    # Python 2: "unicode" is built-in
+    unicode
+except NameError:
+    unicode = str
+
+def uncompress_ipv6(ipv6):
+    addr = unicode(ipv6.split('IPv6:').pop())
```

Comment:
nit: imho ipv6.replace('IPv6:', '') is more readable

Reply:
Fixed